src/main/java/com/duckblade/osrs/toa/TombsOfAmascutConfig.java

```diff
@@ -632,6 +636,52 @@ default SplitsMode splitsOverlay()
 		return SplitsMode.OFF;
 	}
 
+	@ConfigSection(
+		name = "Akkha",
+		description = "Configuration for Akkha boss room.",
+		closedByDefault = true,
+		position = 800
+	)
+	String SECTION_AKKHA = "sectionAkkha";
```

Comment:
This section should be directly below Path of Het, ideally.

Reply:
Tbh I think the whole config could be reordered and cleaned up (e.g. some sections are opened by default while others are closed, it looks messy imo). Reordering won't break anyone's config as long as the keyNames are preserved, right?

Lmk if that's ok and I'll reorder it.

Reply:
Will reorder if/when the config PR is merged